metaflow/plugins/kubernetes/kubernetes.py

```diff
@@ -280,10 +282,14 @@ def create_job(
         for name, value in env.items():
             job.environment_variable(name, value)
 
-        annotations = {
-            "metaflow/user": user,
-            "metaflow/flow_name": flow_name,
-        }
+        annotations = self._get_annotations(annotations)
+        annotations.update(
+            {
+                "metaflow/user": user,
+                "metaflow/flow_name": flow_name,
+            }
+        )
+
         if current.get("project_name"):
```

Comment:
`_get_annotations()` is always being passed an implicit `None` here as `kubernetes.launch_job()` in `kubernetes_cli.py` does not pass annotations. Could just call without an input (seems to be an oversight with the labels as well).

consider if it would be cleaner to lump the system annotations to be defined inside `_get_annotations` as well

Reply:
done